vault/lang/lang.de.inc

```diff
@@ -39,7 +39,7 @@ $MusselConfig['lang']['file']='Datei';
 $MusselConfig['lang']['filesize_limit_exceeded']='Dateigröße überschritten';
 $MusselConfig['lang']['filetype_blacklisted']='Dateityp auf Blacklist';
 $MusselConfig['lang']['finished']='Fertig';
-$MusselConfig['lang']['generated_by']='Generiert&nbsp;von&nbsp;';
+$MusselConfig['lang']['generated_by']='Generiert&nbsp;von';
```

Comment:
@Maikuolan Do we really need this HTML entity (&nbsp;) here? It is the only line which uses it.